README.md

```diff
@@ -1,2 +1,33 @@
-# hw01
-Repository for first CFSS homework assignment
+# Homework 01: Edit README.md
+## HyungJin Cho
+  [//]: # (Comment: #~###### is a header tag <h1>~<h6>)
+
+Brief Biography | GitHub Workflow
+--- | ---
+In this first section, I'll introduce myself using Markdown | In this second section, I'll report my process on GitHub
+  [//]: # (Comment: --- | --- creates a table)
+
```

Comment:
Good demo of Markdown, but I agree. Perfectly appropriate here, but in future assignments make sure the formatting makes sense given the content.